queue/queue.css

```diff
@@ -0,0 +1,22 @@
+.dashboard, .md-datepicker-input {
+	font-family: Bebas_Neue;
+	font-weight: bold;
+	font-size: 2rem !important;
+	line-height: 2rem;
+}
+
+.clock-time {
+	font-size: 2em;
+}
+
+.inactive-period {
+	opacity: 0.15;
+}
+
+/* Center the datepicker calendar on mobile */
+@media (max-width: 767px) {
+	.md-datepicker-calendar-pane.md-pane-open {
+		left: 50% !important;
+		transform: translateX(-50%) !important;
```

Comment:
ew so many !important.... is this overriding the material design defaults?

Reply:
Yeah. I know. I'm not a huge fan. The AngularJS Material Design datepicker isn't really optimized for mobile for some reason. I added this fix to ensure that it would always be centered on mobile. Without it, the calendar that pops up is slightly off-center, which is even more gross IMO.